kernel/packages/shared/comms/index.ts

```diff
@@ -651,7 +651,10 @@ export async function connect(userId: string) {
               if (context && context.currentPosition) {
                 return context.currentPosition.slice(0, 3)
               }
-            }
+            },
+            maxConnectionDistance: 4,
+            nearbyPeersDistance: 5,
+            disconnectDistance: 5
```

Comment:
should this be `10`?

Reply:
This is somewhat confusing, but 5 is ok.

Since we are using the "rings" mechanisms for distance, this actually means that the disconnect distance is for any peer being in the ring 5, it being anything with distance greater than 160 (10 parcels).

Since we are using the same distance function in lighthouse, this is actually defined here: https://github.com/decentraland/catalyst/blob/c84d4ed0893ebb7caa462f40179f727e1c5a9311/commons/utils/Positions.ts#L1

And yes, this means that eventually we will need to update the catalysts too. But fortunately it is not critical, since the effect we will have until we update the catalysts is that when the peer queries the lighthouse for nearby peers, it will respond with every peer, since the lighthouse will be using the old function which only had up to ring 3. But the peer will then sort the information locally. This would only be problematic if we had thousands of peers on each layer.

If I'm being honest, I don't like how confusing this is, and I'm not sure it is worth the confusion haha. Maybe we can change the distance function to be a simple parcel distance...

Reply:
Thanks for clarifying!